### PR TITLE
mcs/resourcemanager: add metric for TiFlash RU consumption

### LINKED ISSUE #7116: add metric for tiflash ru consumption
## Development Task
For tiflash resource control(https://github.com/pingcap/tiflash/issues/7660), tiflash should report its ru consumption to PD/GAC.

### PR DESCRIPTION
**Pull request by guo-shaoge**
### What problem does this PR solve?

Issue Number: Close #7116

### What is changed and how does it work?

```commit-message
add metric for tiflash ru consumption
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
image

- No code

Code changes

- Has the configuration change
- Has HTTP API interfaces changed (Don't forget to [add the declarative for the new API](https://github.com/tikv/pd/blob/master/docs/development.md#updating-api-documentation))
- Has persistent data change

Side effects

- Possible performance regression
- Increased code complexity
- Breaking backward compatibility

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tiup`](https://github.com/pingcap/tiup):
- Need to cherry-pick to the release branch

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by nolouch — APPROVED** [review]
lgtm

**Review by HuSharp — APPROVED** [review]
LGTM!

File: pkg/mcs/resourcemanager/server/metrics.go [hunk 1]
```diff
@@ -26,6 +26,8 @@ const (
 	readTypeLabel          = "read"
 	writeTypeLabel         = "write"
 	backgroundTypeLabel    = "background"
+	tiflashTypeLabel       = "tiflash"
+	tidbTypeLabel          = "tidb"
```
[the remark this answers is not in the document]
Comment (CabinfeverB) [reply pkg/mcs/resourcemanager/server/metrics.go:30]: Users may wonder how much RU TiKV consumes

[the remark this answers is not in the document]
Comment (HuSharp) [reply pkg/mcs/resourcemanager/server/metrics.go:30]: changed to tp/ap https://github.com/tikv/pd/pull/7135

**Pull request merged**